lightning/src/ln/channel.rs

```diff
@@ -281,6 +281,17 @@ impl HTLCCandidate {
 	}
 }
 
+/// Information needed for constructing an invoice route hint for this channel.
+#[derive(Clone)]
+pub struct CounterpartyForwardingInfo {
+	/// Base routing fee in millisatoshis.
+	pub fee_base_msat: u32,
+	/// Amount in millionths of a satoshi the channel will charge per transferred satoshi.
+	pub fee_proportional_millionths: u32,
+	/// The number of blocks to subtract from incoming HTLC cltv_expiry values.
```

Comment:
nit: if other comments come up, can you reword this as its pretty unclear to me? The wording for the similar field in UnsignedChannelUpdate is `/// The number of blocks to subtract from incoming HTLC cltv_expiry values` though that could probably also use a "to calculate the outbound HTLC cltv_expiry value" and a "minimum" somewhere in it.

Reply:
I stole the wording word-for-word from the wording you quoted^, lol. But, updated the comment in both places.

Reply:
I'm more of the mind of having a summary comment in one place and more detailed when where warranted. Otherwise, the documentation is at risk of going out of sync.

Reply:
Adapted to @jkczyz's suggestion!